### ISSUE TITLE
XLSX protected data not preserved in 1.18.0

### ISSUE THREAD

**Issue by lionnel8** [PHPOffice/PhpSpreadsheet#2301]
This is:

```
- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
Protected data is preserved and stays hidden like in version 1.17.1 and older.
![1 17 1](https://user-images.githubusercontent.com/33231575/133739282-c0dad2a7-c340-4462-823a-fd0a79fd7fea.PNG)

### What is the current behavior?
Protected data and hidden formulas become visible.
![1 18 0](https://user-images.githubusercontent.com/33231575/133739374-5fff688d-61cc-4006-a40e-e117870ff9d1.PNG)

### What are the steps to reproduce?
The source file only needs to be loaded and saved to break the protected data.

```php
$srcFile = MAddPath(PrivateDocumentGlobal::getUploadFolderDisc(), self::$file);

$tmpFile = MAddPath($GLOBALS["web_root_disc"], "data/tmp", pathinfo(self::$file, PATHINFO_FILENAME) . "-prevyplneno." . pathinfo(self::$file, PATHINFO_EXTENSION));

$spreadsheet = \PhpOffice\PhpSpreadsheet\IOFactory::load($srcFile);

$writer = new PhpOffice\PhpSpreadsheet\Writer\Xlsx($spreadsheet);
$writer->save($tmpFile);
```

### Which versions of PhpSpreadsheet and PHP are affected?
PhpSpreadsheet 1.18.0 and PHP 7.4.14. Older versions of PhpSpreadsheet, which I have tested until 1.15.0, behave correctly.

**Comment by oleibman** [comment]
Is it possible to upload a spreadsheet which demonstrates this problem?

**Comment by lionnel8** [comment]
Sure, here it is: [template.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/7185743/template.xlsx)

**Comment by oleibman** [comment]
Thank you for the upload. Unfortunately, I am not able to duplicate your problem. When I run your code against release 18.0 (and against master and 17.1 and 14.1), the output spreadsheet displays exactly the same as the uploaded spreadsheet in Excel.

**Comment by lionnel8** [comment]
This is kinda strange. I tested that once again, now locally (PHP 7.4.3) on clear project with just PhpSpreadsheet installed and results of expressions are no longer visible in the sheet, however expressions are still visible in the upper bar when some cell is selected.

1.18.0
![1 18 0_2](https://user-images.githubusercontent.com/33231575/133798301-6d47a13d-ac28-4d3c-bc3e-aa733f2af9f3.png)

1.17.1
![1 17 1_2](https://user-images.githubusercontent.com/33231575/133798334-b20a1c07-595a-4db1-8e1f-22c6193152fb.png)

**Comment by oleibman** [comment]
I agree that 18.0 is showing the formulas. However, master hides them, as desired. If you can test with master, please try. I will try to identify the change that fixed the problem.

**Comment by lionnel8** [comment]
Tested locally with dev-master and everything looks OK, except few cells having larger font than the original file. I'll try on the server on Monday and let you know. Thank you for your effort.

**Comment by lionnel8** [comment]
Tried on server with dev-master, protected data is OK and formulas are no longer visible. 👍 The only flaw is some texts are larger and overflowing, which was not present in any older version I tested. 

![image](https://user-images.githubusercontent.com/33231575/134035446-1b178d70-0a1a-46b4-b006-5688abf40a99.png)

**Comment by oleibman** [comment]
Agree that the formatting problem is not present in 18.0, is present in master. No idea what the problem is.

**Comment by oleibman** [comment]
The problem appears to be in sharedStrings.xml. Master is specifying Calibri font-size 11 for some strings, while prior versions are specifying Arial CE with various font sizes.

**Comment by oleibman** [comment]
This appears to have been an oversight in the new namespace-handling logic. I hope to have a fix available today or tomorrow.

**Comment by oleibman** [comment]
I want to think about my fix a little more, so probably won't merge it for at least a week. If you can test against 2303, please do so. Otherwise, just wait till it's merged.

**Comment by oleibman** [comment]
Fixed in [1.19.0](https://github.com/PHPOffice/PhpSpreadsheet/releases/tag/1.19.0)
